src/main/capi/scalar_function-c.cpp

```diff
@@ -42,9 +42,12 @@ duckdb::ScalarFunction &GetCScalarFunction(duckdb_scalar_function function) {
 	return *reinterpret_cast<duckdb::ScalarFunction *>(function);
 }
 
-unique_ptr<FunctionData> BindCAPIScalarFunction(ClientContext &context, ScalarFunction &bound_function,
+unique_ptr<FunctionData> BindCAPIScalarFunction(ClientContext &, ScalarFunction &bound_function,
                                                 vector<unique_ptr<Expression>> &arguments) {
 	auto &info = bound_function.function_info->Cast<CScalarFunctionInfo>();
+	if (bound_function.HasVarArgs()) {
+		bound_function.arguments.assign(arguments.size(), bound_function.varargs);
```

Comment:
I'm still confused why this is necessary - other varargs functions don't need this, shouldn't the function overloading handle this?

Reply:
Yes, I wasn't aware. I removed it. 👍